ios/ReactNativePayments.m

```diff
@@ -351,24 +352,26 @@ - (void)setRequiredShippingAddressFieldsFromOptions:(NSDictionary *_Nonnull)opti
 {
     // Request Shipping
     if (options[@"requestShipping"]) {
-        self.paymentRequest.requiredShippingAddressFields = PKAddressFieldPostalAddress;
+        //self.paymentRequest.requiredShippingAddressFields = PKAddressFieldPostalAddress;
```

Comment:
fix all these and update everything to use contact field instead of address fields.